subprojects/internal-android-performance-testing/build.gradle.kts

```diff
@@ -1,6 +1,7 @@
 plugins {
     id("gradlebuild.internal.java")
     application
+    id("gradlebuild.dependency-modules")
```

Comment:
I am surprised you need to apply this plugin here explicitly. Doesn't the `gradlebuild.internal.java` plugin apply this?

Reply:
This is a leftover in the development. Removed.